[2023-05-26T15:48:42.789+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL.add_data scheduled__2023-05-25T00:00:00+00:00 [queued]>
[2023-05-26T15:48:42.802+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL.add_data scheduled__2023-05-25T00:00:00+00:00 [queued]>
[2023-05-26T15:48:42.802+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-05-26T15:48:42.802+0000] {taskinstance.py:1283} INFO - Starting attempt 24 of 29
[2023-05-26T15:48:42.802+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-05-26T15:48:42.823+0000] {taskinstance.py:1303} INFO - Executing <Task(PostgresOperator): add_data> on 2023-05-25 00:00:00+00:00
[2023-05-26T15:48:42.831+0000] {standard_task_runner.py:55} INFO - Started process 1125 to run task
[2023-05-26T15:48:42.835+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'ETL', 'add_data', 'scheduled__2023-05-25T00:00:00+00:00', '--job-id', '291', '--raw', '--subdir', 'DAGS_FOLDER/refresh_token_DAG.py', '--cfg-path', '/tmp/tmp7s24nusk']
[2023-05-26T15:48:42.836+0000] {standard_task_runner.py:83} INFO - Job 291: Subtask add_data
[2023-05-26T15:48:42.908+0000] {task_command.py:388} INFO - Running <TaskInstance: ETL.add_data scheduled__2023-05-25T00:00:00+00:00 [running]> on host 05172cfd5782
[2023-05-26T15:48:43.078+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=fox
AIRFLOW_CTX_DAG_ID=ETL
AIRFLOW_CTX_TASK_ID=add_data
AIRFLOW_CTX_EXECUTION_DATE=2023-05-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=24
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-25T00:00:00+00:00
[2023-05-26T15:48:43.079+0000] {sql.py:254} INFO - Executing: 
            insert into spotify_data (played_at, artist, name_track, timestamps) values 
            ('[2023-05-25T09:21:02.777Z', '2023-05-25T09:21:02.777Z']
            ['Kataklysm',
            'Guillotine',
            '2023-05-25'
            )
            
        
[2023-05-26T15:48:43.088+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-05-26T15:48:43.510+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-05-26T15:48:43.514+0000] {sql.py:375} INFO - Running statement: 
            insert into spotify_data (played_at, artist, name_track, timestamps) values 
            ('[2023-05-25T09:21:02.777Z', '2023-05-25T09:21:02.777Z']
            ['Kataklysm',
            'Guillotine',
            '2023-05-25'
            )
            
        , parameters: None
[2023-05-26T15:48:43.516+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 266, in execute
    **extra_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 349, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 380, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.SyntaxError: syntax error at or near "]"
LINE 3: ...   ('[2023-05-25T09:21:02.777Z', '2023-05-25T09:21:02.777Z']
                                                                      ^

[2023-05-26T15:48:43.527+0000] {taskinstance.py:1326} INFO - Marking task as UP_FOR_RETRY. dag_id=ETL, task_id=add_data, execution_date=20230525T000000, start_date=20230526T154842, end_date=20230526T154843
[2023-05-26T15:48:43.545+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 291 for task add_data (syntax error at or near "]"
LINE 3: ...   ('[2023-05-25T09:21:02.777Z', '2023-05-25T09:21:02.777Z']
                                                                      ^
; 1125)
[2023-05-26T15:48:43.572+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-05-26T15:48:43.599+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
